pdm/models/pip_shims.py

```diff
@@ -30,17 +30,13 @@
 )
 from pip._internal.req.req_file import parse_requirements
 from pip._internal.utils import logging as pip_logging
+from pip._internal.utils.compatibility_tags import get_supported
 from pip._internal.utils.filesystem import directory_size, file_size, find_files
 from pip._internal.utils.hashes import FAVORITE_HASH, STRONG_HASHES
 from pip._internal.utils.temp_dir import global_tempdir_manager
 from pip._internal.utils.urls import path_to_url, url_to_path
 from pip._internal.vcs.versioncontrol import VcsSupport
 
-try:
-    from pip._internal.utils.compatibility_tags import get_supported
-except ImportError:
-    from pip._internal.pep425tags import get_supported
-
```

Comment:
The new import path was introduced in pip 20.0. Perhaps a `pip.__version__` check is needed?

Reply:
I agree to just remove the import guard as the proposed change.